src/wandbot/apps/zendesk/__main__.py

```diff
@@ -78,13 +77,9 @@ def format_response(response: str) -> str:
         str: The formatted response.
     """
 
-    response = str(response)
-    max_length = 2000
-    # Truncate the response to a maximum length of 2000 characters
-    if len(response) > max_length:
-        response = response[:max_length] + "..."
-    # Append a signature at the end of the response
-    return response + "\n\n-WandBot 🤖"
+    responseStr = str(response)
```

Comment:
- How are you handling long repsonse now? won't there be an error?
- Fix casing to lower case here and elsewhere, I guess we only use camel case for classes (and types maybe I guess)

Reply:
Hi @morganmcg1!

Thank you so much for looking into it, I have tested and tried it on my side. The character limit for a Zendesk comment is 65,535 characters which is more than the max length of Wandbot's response. If you would like me to I can still set a size check just incase :)

Reply:
I've also edited the description above to share how the script was tested.

Reply:
Closed #46! :)